lib/config/presets/internal/replacements.ts

```diff
@@ -38,6 +39,54 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'emotion-10-to-11': {
+    description:
+      'various unscoped emotion v10 packages became renamed & scoped in v11',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=11.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['@emotion/core', 'emotion-theming'],
+        replacementName: '@emotion/react',
+        replacementVersion: '11.0.0',
+      },
+      {
+        matchCurrentVersion: '>=11.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['emotion', 'create-emotion'],
+        replacementName: '@emotion/css',
+        replacementVersion: '11.0.0',
+      },
+      {
+        matchCurrentVersion: '>=11.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['emotion-server', 'create-emotion-server'],
+        replacementName: '@emotion/server',
+        replacementVersion: '11.0.0',
+      },
+      {
+        matchCurrentVersion: '>=11.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['babel-plugin-emotion'],
+        replacementName: '@emotion/babel-plugin',
+        replacementVersion: '11.0.0',
+      },
+      {
+        matchCurrentVersion: '>=11.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['eslint-plugin-emotion'],
+        replacementName: '@emotion/eslint-plugin',
+        replacementVersion: '11.0.0',
+      },
+      {
+        matchCurrentVersion: '>=11.0.0',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['jest-emotion'],
+        replacementName: '@emotion/jest',
+        replacementVersion: '11.0.0',
```

Comment:
There's a bug at the moment where these rules get filtered out prematurely because despite their package name change, the version isn't changing (`^11.0.0` to `^11.0.0`). Thanks to @JamieMagee for pointing that out.

> Looks like it's added here: https://github.com/renovatebot/renovate/blob/368903ab814f5b19117eaab6c2cb33382be6cad6/lib/workers/repository/process/lookup/index.ts#L150-L160
And then filtered out here: https://github.com/renovatebot/renovate/blob/368903ab814f5b19117eaab6c2cb33382be6cad6/lib/workers/repository/process/lookup/index.ts#L360-L367

Regardless of that bug, given the packages are functionally identical (with only a name change), would it make sense to change the `replacementVersion` to be a patch or minor higher? That way there would be a tangible change in the dependency along with the name change? Would it make sense to pick the lowest or latest `11.x` for each one?

Reply:
I created a bug to track this #14386. It appears to be a simple fix, but I am working on #6562 right now.

I would be okay with putting `11.1.0` as a temporary workaround. Just make sure to document it with a comment to change it once #14386 is resolved.